### PR TITLE
Insight

### PR DESCRIPTION
**Pull request by adeptofvoltron**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -35,4 +35,4 @@ DISTRIBUTION OF THIS DOCUMENT DOES NOT CREATE AN ATTORNEY-CLIENT RELATIONSHIP. C
 COMMONS PROVIDES THIS INFORMATION ON AN "AS-IS" BASIS. CREATIVE COMMONS MAKES NO
 WARRANTIES REGARDING THE USE OF THIS DOCUMENT OR THE INFORMATION OR WORKS PROVIDED HEREUNDER,
 AND DISCLAIMS LIABILITY FOR DAMAGES RESULTING FROM THE USE OF THIS DOCUMENT OR THE
-INFORMATION OR WORKS PROVIDED HEREUNDER.
\ No newline at end of file
+INFORMATION OR WORKS PROVIDED HEREUNDER.
```
Comment (pradzikowski) [inline LICENSE:38]: I cant see any changes ?

File: src/RegexVerifier.php [hunk 2]
```diff
@@ -39,7 +39,7 @@ protected function doVerifyDomain(string $domain): int
     private function pregList(array $list, string $domain): bool
     {
         foreach ($list as $pattern) {
-            if (true == preg_match("/$pattern/", $domain)) {
+            if (true === preg_match("/$pattern/", $domain)) {
```
Comment (pradzikowski) [inline src/RegexVerifier.php]: `preg_match` returns int.

**Review by pradzikowski — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (adeptofvoltron) [reply LICENSE:38]: new line in the end

**Comment by adeptofvoltron** [comment]
@pradzikowski all done

File: LICENSE (hunk 1 above)
Comment (pradzikowski) [reply LICENSE:38]: :ok_hand:

**Review by pradzikowski — APPROVED** [review]

**Pull request merged**